docs/lib/content/commands/npm-doctor.md

```diff
@@ -21,68 +20,57 @@ standalone tool, but it does have some basic requirements that must be met:
   globally), exist and can be written by the current user.
 + The npm cache exists, and the package tarballs within it aren't corrupt.
 
-Without all of these working properly, npm may not work properly.  Many
-issues are often attributable to things that are outside npm's code base,
+Without all of these working properly, npm may not work properly.
+Many issues are often attributable to things that are outside npm's code base,
 so `npm doctor` confirms that the npm installation is in a good state.
 
-Also, in addition to this, there are also very many issue reports due to
-using old versions of npm. Since npm is constantly improving, running
-`npm@latest` is better than an old version.
+Also, in addition to this, there are also very many issue reports due to using old versions of npm.
+Since npm is constantly improving, running `npm@latest` is better than an old version.
 
-`npm doctor` verifies the following items in your environment, and if
-there are any recommended changes, it will display them.  By default npm
-runs all of these checks. You can limit what checks are ran by
-specifying them as extra arguments.
+`npm doctor` verifies the following items in your environment, and if there are any recommended changes, it will display them.
+By default npm runs all of these checks.
+You can limit what checks are ran by specifying them as extra arguments.
 
 #### `Connecting to the registry`
 
 By default, npm installs from the primary npm registry,
-`registry.npmjs.org`.  `npm doctor` hits a special connection testing
-endpoint within the registry. This can also be checked with `npm ping`.
-If this check fails, you may be using a proxy that needs to be
-configured, or may need to talk to your IT staff to get access over
+`registry.npmjs.org`.
+`npm doctor` hits a special connection testing endpoint within the registry.
+This can also be checked with `npm ping`.
+If this check fails, you may be using a proxy that needs to be configured, or may need to talk to your IT staff to get access over
 HTTPS to `registry.npmjs.org`.
```

Comment:
I think it missed an inline here.

Reply:
The unwrapping I'm doing doesn't handle capital letters at the beginning of a line, I just reviewed those manually. I ran out of time for the week and decided to post as is.

I'll do a few more passes through the same files to try to catch things like this. I may have to think about the right way to approach it. Perhaps any line that doesn't end with a `.`/`:`/`!`/`?` can be unwrapped... I already realized that lines that start with `` ` `` or that end with `,` still deserve to be unwrapped.